### PR TITLE
Add ssx example processed data from I24

### PR DESCRIPTION
**Pull request by jbeilstenedmands**
Adds small example `imported.expt` files and `strong.refl` files for testing indexing of ssx images.

### CODE REVIEW

File: dials_data/definitions/ssx_CuNiR_test_data.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+name: SSX CuNiR test data
```
Comment (Anthchirp) [inline dials_data/definitions/ssx_CuNiR_test_data.yml:1]: Recommend dropping `test` and `data` from the file name, you could use a `_processed` suffix as in https://github.com/dials/data/blob/master/dials_data/definitions/mpro_x0305_processed.yml

Name could be
```suggestion
name: DIALS output of processing a SSX CuNiR data set
```
to distinguish from raw data

File: dials_data/definitions/ssx_CuNiR_test_data.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+author: James Beilsten-Edmands (2021)
+license: CC-BY 4.0
+description: >
+  Example processed SSX data on CuNiR, taken on I24 beamline at Diamond
+  Light Source, courtesy of Robin Owen and Mike Hough.
+  5 images were processed with dials v3.7.3, with and without a reference
```
Comment (Anthchirp) [inline dials_data/definitions/ssx_CuNiR_test_data.yml:7]:
```suggestion
  5 images were processed with DIALS v3.7.3, with and without a reference
```

**Review by Anthchirp — CHANGES_REQUESTED** [review]
Is the original data available anywhere? Even if it is not publicly available it would be great to identify the original data, eg. by file path (cf. [mpro](https://github.com/dials/data/blob/master/dials_data/definitions/mpro_x0305_processed.yml)) so that it can be found again in the future.

**Comment by jbeilstenedmands** [comment]
Thanks, there's no zenodo link or ispyb link, but I'll see if I can find where the data are located on disk at DLS.
I'll update the name too.

**Pull request merged**